ocaml/idl/datamodel.ml

```diff
@@ -5587,6 +5590,33 @@ let storage_plugin =
      ])
     ()
 
+let lvhd_enable_thin_provisioning = call
+   ~name:"enable_thin_provisioning"
+   ~in_oss_since:None
+   ~in_product_since:rel_dundee
+   ~allowed_roles:_R_POOL_ADMIN
+   ~params:[
+     Ref _sr, "SR", "The LVHD SR to upgrade to being thin-provisioned.";
+     Int, "initial_allocation", "The initial amount of space to allocate to a newly-created VDI in bytes";
```

Comment:
The XenAPI standard is bytes, not MB or MiB or KB or KiB or GB or GiB-- the
implementation will have to convert.

Standardising on bytes has saved us *loads* of problems in the past,
compared to the previous implementation...

On Fri, 21 Aug 2015 16:23 Si Beaumont <notifications@github.com> wrote:

> In ocaml/idl/datamodel.ml
> <https://github.com/xapi-project/xen-api/pull/2354#discussion_r37645013>:
>
> > @@ -5587,6 +5590,33 @@ let storage_plugin =
> >       ])
> >      ()
> >
> > +let lvhd_enable_thin_provisioning = call
> > +   ~name:"enable_thin_provisioning"
> > +   ~in_oss_since:None
> > +   ~in_product_since:rel_dundee
> > +   ~allowed_roles:_R_POOL_ADMIN
> > +   ~params:[
> > +     Ref _sr, "SR", "The LVHD SR to upgrade to being thin-provisioned.";
> > +     Int, "initial_allocation", "The initial amount of space to allocate to a newly-created VDI in bytes";
>
> I was just told that they want to use MB here, not bytes. :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xapi-project/xen-api/pull/2354/files#r37645013>.
>